views/partials/search-form-fields.html

```diff
@@ -96,7 +120,10 @@
   </div>
 
   <div class="filter">
-    <label for="has_published_results">Study results</label>
+    <label for="has_published_results"
+           class="hastip"
+           title="If the trial has posted results on ClinicalTrials.gov"
```

Comment:
This isn't specific to CT.gov; this is if we have found the trial's results at all, wherever that comes from (could even be from a data donation).